go/oasis-node/cmd/common/flags/flags.go

```diff
@@ -100,6 +112,9 @@ func init() {
 
 	EntityFlags.StringP(cfgEntity, "e", "", "Path to directory containing entity private key and descriptor")
 
+	SignerFlags.StringP(cfgSigner, "s", "", "Type of signer to use. Options: [file, ledger]")
+	SignerFlags.String(cfgFileSignerDir, "", "Path to file containing private key")
```

Comment:
The description is incorrect (it should point to the directory containing the private keys). Also it should default to the data directory as otherwise this breaks existing use cases.